data/mods/Magiclysm/recipes/dragon_black.json

```diff
@@ -2,7 +2,6 @@
   {
     "type": "recipe",
     "result": "black_dragon_tanning_hide",
-    "byproducts": [ [ "dragon_black_scale", 20 ] ],
```

Comment:
please don't remove this, i want you to be able to get scales from the hides.

Reply:
I understand, should I keep the number or is it okay to remove some?